pkg/controller/complianceremediation/complianceremediation_controller.go

```diff
@@ -613,6 +619,70 @@ func (r *ReconcileComplianceRemediation) verifyAndCompleteMC(obj *unstructured.U
 	return nil
 }
 
+// Process kubeletconfig remediation
+func (r *ReconcileComplianceRemediation) verifyAndCompleteKC(obj *unstructured.Unstructured, rem *compv1alpha1.ComplianceRemediation) error {
```

Comment:
it would be good to start testing this with unit tests

Reply:
The unit test is all ready, just need the e2e tests